app/src/main/java/com/github/wnder/GuessLocationActivity.java

```diff
@@ -64,12 +78,22 @@
  */
 @AndroidEntryPoint
 public class GuessLocationActivity extends AppCompatActivity implements OnMapReadyCallback, MapboxMap.OnMapClickListener, MapboxMap.OnCameraMoveListener {
+
+    @Inject
+    public FirebasePicturesDatabase db;
+
+    private int zoomAnimationTime;
```

Comment:
Final variable maybe? And if adding "final" for this variable is better shouldn't we just use the android.R.integer.config_shortAnimTime instead of that variable?

Reply:
I guess that's what I already do in the constructor thanks to "getResources()", which needs a context, so we can't do it in the fields.